### PR TITLE
DEV-26343 Add user create notes with federated auth enabled 

### PR DESCRIPTION
**Pull request by tkrisztian95**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1581,7 +1581,12 @@ so the `+` character encoded to `%2B` will remain valid. Alternatively, the `+`
 - License type can be one of 'named' or 'concurrent' (optional)
 - Must have `UserAndRoles.editUser` privileges to perform this request
 
-**Note:** Extra caution should be taken when there are no configured default roles. This would result in no roles being automatically assigned.
+**Extra caution should be taken, in cases:** 
+- There are no configured default roles, this would result in no roles being automatically assigned to new users.
+- Federated authentication is enabled:
+  - User creation happens automatically with Federated Authentication.
+  - Only users created by the identity provider can sign in to the Acrolinx Dashboard.
```
Comment (george-haddad) [inline apiary.apib:1588]: This felt right to add hyphen for sign-in

```suggestion
  - Only users created by the identity provider can sign-in to the Acrolinx Dashboard.
```

File: apiary.apib [hunk 2]
```diff
@@ -1581,7 +1581,12 @@ so the `+` character encoded to `%2B` will remain valid. Alternatively, the `+`
...
+  - Users created through the API, should only be used for automation purposes such as interact with Platform API but can’t get access to the Dashboard.
```
Comment (george-haddad) [inline apiary.apib:1589]: This word "should" implies that it also _can_ get access to the dashboard but the user shouldn't. If the case is that they actually cannot do that then we can just `Users created through the API can only be used for automation purposes such as interactions with the Platform API`

Comment (tkrisztian95) [reply apiary.apib:1589]: Thanks!

**Review by george-haddad — APPROVED** [review]
This looks good for me.

**Pull request merged**